### PR TITLE
Fix hide Originally Airs, when it's not available in the indexer.

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/tv.py [hunk 1]
```diff
@@ -1103,13 +1103,14 @@ def load_from_indexer(self, tvapi=None):
...
         self.imdbid = getattr(indexed_show, 'imdb_id', '')
 
-        if getattr(indexed_show, 'airs_dayofweek', None) is not None and getattr(indexed_show, 'airs_time', None) is not None:
-            self.airs = indexed_show['airs_dayofweek'] + ' ' + indexed_show['airs_time']
+        if getattr(indexed_show, 'airs_dayofweek', '') and getattr(indexed_show, 'airs_time', ''):
+            self.airs = '{airs_day_of_week} {airs_time}'.format(airs_day_of_week=indexed_show['airs_dayofweek'],
+                                                                airs_time=indexed_show['airs_time'])
 
-        if self.airs is None:
+        if self.airs is '':
```
Comment (fernandog) [inline medusa/tv.py]: Do we need this if?

Comment (fernandog) [inline medusa/tv.py]: my point: if is already '' why set to ''?

Comment (ratoaq2) [inline medusa/tv.py]: Yeah, this could be safely removed

**Comment by fernandog** [comment]
approved

**Comment by fernandog** [comment]
Fixes https://github.com/pymedusa/Medusa/issues/1352

**Pull request merged**